lab-abigail/controller/food-controller.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+const createError = require('http-errors');
+const Food = require('../model/food');
+
+module.exports = exports = {};
+
+const URL = `${__dirname}/../data`;
+
+exports.createItem = function(req, res, food) {
+
+  if(!food) return Promise.reject(createError(400, 'bad requst'));
+
+  new Food(req.body).save()
+  .then(food => {
+    console.log(food);
+    res.json(food);
+  })
+  .catch(err => res.status(400).send(err.message));
+};
+
+
+exports.updateItem = function(req, res, id, food) {
+
+  if(!id) return Promise.reject(createError(404, 'not found'));
+
+  Food.findByIdAndUpdate(id, food, {new: true})
```

Comment:
Good job figuring out the placement of ``{new: true}`` to update this properly! 👍